protocol/prottest/memstore.go

```diff
@@ -1,8 +1,8 @@
-// Package memstore provides a Store implementation that
+// MemStore is a Store implementation that
 // keeps all blockchain state in memory.
 //
 // It is used in tests to avoid needing a database.
```

Comment:
Should get rid of this package comment if we're moving it into prottest. It no longer describes the package but the type.